Data/Sys/GameSettings/HC4.ini

```diff
@@ -0,0 +1,14 @@
+# HC4P01, HC4E9Z, HC4P9Z - Crunchyroll Channel
```

Comment:
Where does "HC4P01" come from?

Reply:
Probably from outdated wiki information. (Sometimes people seemingly guessed the publisher ID of downloadable titles because GameTDB doesn't provide that information.)

Reply:
Weird, I have taken the GameID's from the wiki but I can't find "HC4P01" anywhere now. No idea how that happend. I will remove it.